### DISCUSSION TITLE
Hide information that appears when you open terminal

### DISCUSSION THREAD

**Discussion by ghost** [CachyOS/linux-cachyos#86] [Q&A] [answered]
Every time I open the terminal it appears a ton of information (OS, host, kernel, Display, WM, Theme, Icons...) and an image (a big C) in the terminal. I would like all of that to disappear.

**Comment by sirlucjan** [comment]
How does this relate to the kernel?

**Comment by ptr1337** [comment, marked as answer]
Hi!

You can simply comment following lines:
https://github.com/CachyOS/cachyos-fish-config/blob/main/cachyos-config.fish#L159-L162

This file is located in `/usr/share/cachyos-fish-config/cachyos-config.fish`

Please mark this as solved if it is working correclty for you

**Reply by ghost** [reply]
The aliases work (I did not know I have to put them in config.fish). But I cannot solve the problem of showing fastfetch output in the terminal.  When I go to /usr/share/cachyos-fish-config/cachyos-config.fish, I see the lines you say:
```
## Run fastfetch if session is interactive
if status --is-interactive
   fastfetch
end
```
But I do not know how to *comment* them. What should I put in those lines to have a clean terminal?

**Comment by sirlucjan** [comment]
Just put # before.
